### PR TITLE
feat: Added ec2 host (dedicated)

### PR DESCRIPTION
**Pull request by zagronitay**

### CODE REVIEW

**Review by disq — APPROVED** [review]

**Comment by disq** [comment]
I approved before the migrations? 😮

File: resources/services/ec2/hosts.go [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+			{
+				Name:        "tags",
+				Description: "Any tags assigned to the Dedicated Host.",
+				Type:        schema.TypeJSON,
+				Resolver:    client.ResolveTags,
+			},
+		},
+		Relations: []*schema.Table{
+			{
+				Name:        "aws_ec2_host_available_capacity_available_instance_capacity",
```
Comment (disq) [inline resources/services/ec2/hosts.go:157]: Maybe just `aws_ec2_host_available_instance_capacity`?

**Review by disq — CHANGES_REQUESTED** [review]

**Review by disq — APPROVED** [review]

File: docs/tables/aws_ec2_hosts.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+| ------------- | ------------- | -----  |
+|account_id|text|The AWS Account ID of the resource.|
+|region|text|The AWS Region of the resource.|
+|arn|text|The Amazon Resource Name (ARN) for the dedicated host.|
+|allocation_time|timestamp without time zone|The time that the Dedicated Host was allocated.|
+|allows_multiple_instance_types|text|Indicates whether the Dedicated Host supports multiple instance types of the same instance family|
+|auto_placement|text|Whether auto-placement is on or off.|
+|availability_zone|text|The Availability Zone of the Dedicated Host.|
+|availability_zone_id|text|The ID of the Availability Zone in which the Dedicated Host is allocated.|
+|available_capacity_available_vcpus|integer|The number of vCPUs available for launching instances onto the Dedicated Host.|
```
Comment (roneli) [inline docs/tables/aws_ec2_hosts.md:15]: available_vcpus

File: docs/tables/aws_ec2_host_available_instance_capacity.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+
+# Table: aws_ec2_host_available_instance_capacity
```
Comment (roneli) [inline docs/tables/aws_ec2_host_available_instance_capacity.md:2]: maybe a json array?

**Review by roneli — COMMENTED** [review]
LGTM, had one naming nit, maybe convert available capacity to a json~ not sure tbh there

**Review by roneli — APPROVED** [review]

**Pull request merged**